tests/parm/input_natl_c192.nml.IN

```diff
@@ -0,0 +1,366 @@
+ &amip_interp_nml
```

Comment:
please remove this amip_interp_nml namelist, it's confirmed by GFDL they don't need to be listed here.

Reply:
Thanks, @junwang-noaa! These changes have been made accordingly.